parl/env/multiagent_simple_env.py

```diff
@@ -67,6 +68,11 @@ def get_shape(self, input_space):
                     '[Error] shape is {}, not Box or Discrete or MultiDiscrete'
                     .format(input_space.shape))
                 raise NotImplementedError
+
+    logger.warning(
+        'the `MAenv` from `parl.env.multiagent_simple_env` is deprecated since parl 2.0.4 and will be removed in parl 3.0. \n \
+        We recomend you to use `from parl.env.multiagent_env import MAenv` instead, which provides arguments for continuous_actions.'
```

Comment:
provides arguments for continuous_actions. ->
supports continuous control.

Reply:
done